studio/helpers.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime, timezone
+
+from django.contrib.auth.models import User
+from django.db import transaction
+
+
+def do_delete_account(user_id):
```

Comment:
```suggestion
def delete_account(user_id):
```
looks cleaner to me. Why the `do`?

Reply:
There already exists a view method with the name delete_account